cmd/cstor-pool-mgmt/pool/pool.go

```diff
@@ -107,10 +126,19 @@ func createPoolBuilder(cStorPool *apis.CStorPool) []string {
 
 	poolNameUID := string(PoolPrefix) + string(cStorPool.ObjectMeta.UID)
 	createAttr = append(createAttr, poolNameUID)
-	// To generate mirror disk0 disk1 mirror disk2 disk3 format.
-	for i, disk := range cStorPool.Spec.Disks.DiskList {
-		if cStorPool.Spec.PoolSpec.PoolType == "mirrored" && i%2 == 0 {
-			createAttr = append(createAttr, "mirror")
+	if poolType == "" || poolType == "striped" {
```

Comment:
why poolType checking for ""?

Reply:
This was just an assumption that what if type is empty since I have not seen any case in mgmt code to handle striped type. I will remove it